addOns/automation/src/main/java/org/zaproxy/addon/automation/AutomationJob.java

```diff
@@ -123,6 +125,14 @@ public String getSummary() {
         return EMPTY_SUMMARY;
     }
 
+    public boolean isEnabled() {
+        return enabled;
+    }
+
+    public void setEnabled(boolean enabled) {
+        this.enabled = enabled;
+    }
```

Comment:
Could use https://github.com/zaproxy/zaproxy/blob/main/zap/src/main/java/org/zaproxy/zap/utils/EnableableInterface.java ?

Reply:
No benefit for now.